src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -41,17 +33,41 @@ import {
   HAS_GRID_TEMPLATES,
   HAS_REACTIONS_TEMPLATES,
   BACKGROUND_IMG_TEMPLATES,
+  USER_STATUSES,
   DEFAULT_SHOW_SCHEDULE,
+  ONLINE_USER_STATUS,
 } from "settings";
-import "./Venue.scss";
-import { PlayaContainer } from "pages/Account/Venue/VenueMapEdition";
-import { ExtractProps } from "types/utility";
+
 import { IS_BURN } from "secrets";
+
+import { createJazzbar } from "utils/venue";
+import { venueLandingUrl } from "utils/url";
+
+import { VenuePlacementState, VenueTemplate } from "types/venues";
+import { ExtractProps } from "types/utility";
+import { UserStatus } from "types/User";
```

Comment:
coool! Thanks for sorting these imports out! 😍 
could you also place `types/` earlier than `utils/` ? 

```suggestion
import { VenuePlacementState, VenueTemplate } from "types/venues";
import { ExtractProps } from "types/utility";
import { UserStatus } from "types/User";

import { createJazzbar } from "utils/venue";
import { venueLandingUrl } from "utils/url";
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/6db8a73718475d1f8e780610acf3becd6f5e4c3b